src/types/shared.ts

```diff
@@ -22,21 +29,38 @@ export enum TickerEndType {
     unlimited = 'unlimited',
     hardstop = 'hardstop', // currently unsupported
 }
+
+export const tickerEndTypeSchema = z.enum(['unlimited', 'hardstop']);
+
 export enum TickerCountType {
     money = 'money',
     people = 'people',
 }
+
+export const tickerCountTypeSchema = z.enum(['money', 'people']);
+
```

Comment:
@tomrf1 I've left the duplication here for now. We could change `TickerCountType` to be 

```ts
export type TickerCountType = 'money' | 'people'
```

which would allow us to use `zod`'s inferred type, but would mean a few more code changes